scripts/seed.py

```diff
@@ -0,0 +1,210 @@
+# Command to run : python manage.py shell --settings=settings.dev  < scripts/seed.py
+
+from django.conf import settings
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+from allauth.account.models import EmailAddress
+
+from datetime import timedelta
+from faker import Factory
+
+import os
+
+from challenges.models import Challenge, ChallengePhase, DatasetSplit, Leaderboard, ChallengePhaseSplit
+from hosts.models import ChallengeHostTeam, ChallengeHost
+from participants.models import Participant, ParticipantTeam
+
+
+def create_admin_user():
+    username = "admin"
+    password = "password"
+    email = "admin@gmail.com"
+    user = User.objects.create_user(
+        email=email,
+        username=username,
+        password=password,
+        is_staff=True,
+        is_superuser=True,
+    )
+    EmailAddress.objects.create(user=user, email=email, verified=True, primary=True)
+    print "Super user created with \n username: %s \n password: %s" % (username, password)
```

Comment:
Can you use `format` function here to format the string with arguments ? Also there is no need of using new lines. A single line would do.

Also please make sure that you are using format everywhere instead of  writing`%s` or `%d` based print statements.